solidity/random-beacon/deploy/09_deploy_random_beacon_chaosnet.ts

```diff
@@ -0,0 +1,35 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction, DeployOptions } from "hardhat-deploy/types"
+
+const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
+  const { getNamedAccounts, deployments, helpers } = hre
+  const { deployer } = await getNamedAccounts()
+
+  const deployOptions: DeployOptions = {
+    from: deployer,
+    log: true,
+    waitConfirmations: 1,
+  }
+
+  const RandomBeaconChaosnet = await deployments.deploy(
```

Comment:
We could integrate `RandomBeaconChaosnet` contract deployment with the `RandomBeacon` contract deployment https://github.com/keep-network/keep-core/blob/main/solidity/random-beacon/deploy/04_deploy_random_beacon.ts

If we run the deployment right now we will deploy both `RandomBeacon` `RandomBeaconChaosnet`. We should determine a contract to be deployed based on the network we're running the script against. See example here: https://github.com/keep-network/tbtc-v2/blob/6ff3d7b1c7d0a421507037acf3a421c4261dfcdd/solidity/deploy/00_resolve_relay.ts#L21-L26

Reply:
We decided to introduce a flag (`chaosnet`) that decides whether we should deploy something or not.
See: https://github.com/keep-network/keep-core/pull/3470 and https://github.com/keep-network/keep-core/pull/3460.